x-pack/plugins/alerting/server/rule_type_registry.ts

```diff
@@ -436,10 +432,6 @@ export class RuleTypeRegistry {
   public getAllTypes(): string[] {
     return [...this.ruleTypes.keys()];
   }
-
-  public getLatestRuleVersion() {
-    return this.latestRuleVersion;
-  }
 }
```

Comment:
All these `getLatestRuleVersion` and `atestRuleVersion` implementations are leftovers from task execution skip feature that we have already removed.